packages/grafana-ui/src/components/GraphNG/GraphNG.tsx

```diff
@@ -328,7 +328,7 @@ export const GraphNG: React.FC<GraphNGProps> = ({
   );
 };
 
-const mapMouseEventToMode = (event: React.MouseEvent): GraphNGLegendEventMode => {
+export const mapMouseEventToMode = (event: React.MouseEvent): GraphNGLegendEventMode => {
```

Comment:
Think we can move this to new file utils.ts in the GraphNG dir

Reply:
I removed this export for now since we are not using it.  I think the click behavior we want here is opposite to the graph, but we can sort that out in 7.5+